jsonschema/cli.py

```diff
@@ -251,7 +260,24 @@ def load(_):
                 raise _CannotLoadFile()
         instances = ["<stdin>"]
 
-    validator = arguments["validator"](schema)
+    if arguments["base_uri"] is None:
+        resolver = None
+    elif "http:" in arguments["base_uri"] or "https:" in arguments["base_uri"]\
```

Comment:
This behavior doesn't look like it's tested, but also it doesn't seem exactly correct.

There are many schemes (many URIs) that may be provided here -- jsonschema allows any scheme, including user defined ones, and will retrieve them today via `requests`, which itself allows defining handlers for any scheme someone would like.

But the behavior here shouldn't differ across schemes -- what's making us split file out?